lib/types.ts

```diff
@@ -10,27 +10,34 @@ export interface SbReactSDKOptions extends SbSDKOptions {
 }
 
 export type {
-  AlternateObject,
-  Richtext,
-  RichtextInstance,
+  apiPlugin,
```

Comment:
apiPlugin should be a normal export (not type)

Reply:
@alexjoverm this is also exported as a type in the Vue 3 SDK, could this be the cause of the apiPlugin issue?

Reply:
@alexjoverm changed!

The normal export is here:
https://github.com/storyblok/storyblok-react/blob/main/lib/index.ts#L19

Reply:
@fgiuliani perfect!
@Dawntraoz not really. If you remember from the last call we know where the problem is.